app/models/periodic_expense_runner.rb

```diff
@@ -27,12 +27,6 @@ def update_periodic_expense(periodic_expense)
   end
 
   def create_expense(periodic_expense)
-    Expense.create!(
-      name: "#{periodic_expense.name} (#{periodic_expense.period.downcase})",
-      user: periodic_expense.user,
-      periodic_expense: periodic_expense,
-      paid_on: today.to_datetime,
-      amount: periodic_expense.amount
-    )
+    periodic_expense.create_expense.save
```

Comment:
Because I'm using `create` in PeriodicExpense, not `create!`. This way only PeriodicExpenseRunner does changes to the database. Is this ok?